client/src/components/users/UserFilter.vue

```diff
@@ -54,26 +57,24 @@ async function openPopover(event: Event): Promise<void> {
 #select-filter-popover-button {
   position: relative;
   --background: transparent;
-  --color: var(--parsec-color-light-secondary-text);
-  --background-hover: transparent;
+  --color: var(--parsec-color-light-secondary-hard-grey);
+  --color-hover: var(--parsec-color-light-secondary-text);
+  --background-hover: none;
   border-radius: var(--parsec-radius-8);
 
   &::part(native) {
     padding: 0.375rem 0.625rem;
     overflow: visible;
   }
 
-  &:hover {
-    --background: var(--parsec-color-light-secondary-medium);
-  }
-
-  ion-icon {
-    color: var(--parsec-color-light-secondary-grey);
+  .filter-button__icon {
+    color: var(--parsec-color-light-secondary-hard-grey);
     margin-right: 0.5rem;
+    font-size: 1rem;
   }
 
-  &:hover ion-icon {
-    color: var(--parsec-color-light-primary-700);
+  &:hover .filter-button__icon {
+    color: var(--parsec-color-light-secondary-text);
```

Comment:
Hover color does not apply properly to the whole button (behaves fine on workspaces)


https://github.com/user-attachments/assets/19114038-d372-4141-a351-122e399a2445